calico_kubernetes/logutils.py

```diff
@@ -1,17 +1,30 @@
 #!/bin/python
 import logging
 import os
+import pycalico
+
+_log = logging.getLogger(pycalico.__name__)
 
 LOG_DIR = '/var/log/calico/kubernetes'
+ROOT_LOG_FORMAT = '%(asctime)s %(process)d %(levelname)s %(message)s'
+LOG_FORMAT = '%(asctime)s %(process)d %(levelname)s %(filename)s: %(message)s'
+
+def configure_logger(logger, root_logger=False, log_dir=None):
```

Comment:
Could you add a docstring that describes how you expect this to be used, and explains the arguments and their defaults?